packages/edit-post/src/components/layout/index.js

```diff
@@ -141,7 +141,7 @@ function Layout() {
 									</div>
 								) }
 								<SettingsSidebar />
-								<Sidebar.Slot />
+								<PluginComplementaryArea.Slot scope="edit-post/complementary-area" />
```

Comment:
This slot is has the `complementary-area` part already included in the name if the component, does it need to be repeated in the scope or is it enough to leave the name of the page? Like:
```suggestion
								<PluginComplementaryArea.Slot scope="edit-post" />
```